code/modules/modular_computers/computers/item/pda.dm

```diff
@@ -246,6 +246,14 @@
 	if(messenger_app)
 		messenger_app.ringtone = new_ringtone
 
+/obj/item/modular_computer/pda/imprint_id(name = null, job_name = null)
+	. = ..()
+	add_messenger(src)
```

Comment:
Should this be handled by the Messenger app instead?

Reply:
Probably, I made it like that at some point but I ended up reverting it because it was getting too unwieldy to work with, if I remember correctly